tfhe/src/high_level_api/shortints/tests.rs

```diff
@@ -1,8 +1,9 @@
+#![allow(clippy::assign_op_pattern)]
```

Comment:
what is this for ?

Reply:
clippy conplains about doing `let c = a + b`, it would like us to do `a+=b;`

Reply:
thanks